Yoast/wordpress-seo
# SEMrush OAuth integration

## Context


* Introduces the OAuth integration with SEMrush and allows basic authentication and token refreshing.

## Summary


This PR can be summarized in the following changelog entry:

* Adds OAuth for communication with SEMrush API.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Ensure you have set the OAuth provider's `clientSecret` property. (Secret available on request)
* Run the tests, ensure they pass.
* To be continued.

## UI changes

## Documentation

## Quality assurance
